src/functions/schema.ts

```diff
@@ -55,8 +57,22 @@ export interface Dates {
  * Enum for TimePeriod
  */
 export enum TimePeriod {
-  ALL_TIME = 'ALL_TIME',
-  YEAR ='YEAR',
-  MONTH = 'MONTH',
-  WEEK = 'WEEK',
+  ALL_TIME = "ALL_TIME",
+  YEAR ="YEAR",
+  MONTH = "MONTH",
+  WEEK = "WEEK"
+}
+
+/* 
+* interface for timeregistrations
+*/
+
+export interface timeRegistrations {
```

Comment:
```suggestion
/* 
* Interface for time registrations
*/
export interface TimeRegistration {
```

Reply:
First case in interface names should be capital

Reply:
Done